cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -11,10 +11,10 @@ You can use the *Developer* perspective of the {product-title} web console to cr
 
 In the *Developer* perspective:
 
-* Use the *Add* -> *Pipeline* -> *Pipeline Builder* option to create customized pipelines for your application.
+* Use the *Add* -> *Pipeline* -> *Pipeline builder* option to create customized pipelines for your application.
 * Use the *Add* -> *From Git* option to create pipelines using operator-installed pipeline templates and resources while creating an application on {product-title}.
 
-After you create the pipelines for your application, you can view and visually interact with the deployed pipelines in the *Pipelines* view. You can also use the *Topology* view to interact with the pipelines created using the *From Git* option. You need to apply custom labels to a pipeline created using the *Pipeline Builder* to see it in the *Topology* view.
+After you create the pipelines for your application, you can view and visually interact with the deployed pipelines in the *Pipelines* view. You can also use the *Topology* view to interact with the pipelines created using the *From Git* option. You need to apply custom labels to a pipeline created using the *Pipeline builder* to see it in the *Topology* view.
```

Comment:
```suggestion
After you create the pipelines for your application, you can view and visually interact with the deployed pipelines in the *Pipelines* view. You can also use the *Topology* view to interact with the pipelines created using the *From Git* option. You must apply custom labels to a pipeline created using *Pipeline builder* to see them in the *Topology* view.
```

Reply:
Is this resolved? If so, please resolve the conversation.